docs/user-guide/data/work-on-data.md

```diff
@@ -0,0 +1,242 @@
+
+![Single resource view. The resource type in this example is "Page".](../../assets/images/resource-view-with-image.png)*Single resource view. The resource type in this example is "Page" and an image viewer displays the photograph of the manuscript page.*
+
+Everything about the resource is displayed below the media viewer (when there is one). All the information about the resource itself (e.g. type, label, permissions, creator, date of creation, etc.), the data (properties and values), and different functionalities such as editing, deleting, sharing the resource. You can find more information about the functionalities in the following sections.
+
+![Resource toolbar description.](../../assets/images/resource-toolbar.png)
+
+##### Audio file
+
+Supported file types: **mp3**, **wav**
+
+![Audio viewer](../../assets/images/audio-file.png)
+
+##### Image file
+
+Supported file types: **jpeg**, **jp2**, **jpg**, **tiff**, **tif**, **png**
+
+In a resource of type "still image", you're able to draw regions on the image and annotate this region. Usually, a still image resource is used for book pages, photographs, postcards, letters, etc.
+
+When you open a resource that contains an image, it is displayed in a viewer. Several functionalities are accessible from the image viewer, e.g. zoom in/out, copy the IIIF link, replace the image or draw a region on the image.
+
+<!-- ![Image viewer](../assets/images/region_annotation_panel.png)*Image viewer toolbar.* -->
+![Image viewer](../../assets/images/image-file.png)
+
+To access the regions, go to the **annotation** panel. You can **click on an existing region** in the image viewer, the focus of the page will be redirected to the annotation information.
+
+![Access the annotation panel](../../assets/images/access_annotation_panel.png)*Access the annotation panel from the resource toolbar.*
+
+##### Document file
+
+Supported file types: **pdf**, **doc**, **docx**, **xls**, **xlsx**, **ppt**, **pptx**
+
+![PDF viewer](../../assets/images/pdf-file.png)
+
+##### Text file
+
+Supported file types: **csv**, **txt**, **xml**, **xsd**, **xsl**
+
+![Text viewer](../../assets/images/text-doc-file.png)
+
+##### Video file
+
+Supported file types: **mp4**
+
+![Video viewer](../../assets/images/video-file.png)
+
+##### Archive file
+
+Supported file types: **7z**, **gz**, **gzip**, **tar**, **tgz**, **z**, **zip**
+
+![Archive viewer](../../assets/images/zip-file.png)
+
+### Display resource properties
+
+By default, only important properties are shown in the resource viewer. To display them all, click on the "unfold" button in the resource toolbar.
+
+![Show or hide properties.](../../assets/images/show-hide-properties.png)*Show or hide properties from the toolbar.*
+
+To get more information about the property value, i.e. date of creation and author name, you mouse over the value and then on the info button.
```

Comment:
~~you mouse~~ **hover** over the value and then ~~on~~ **over** the info button.